src/asm/fstack.c

```diff
@@ -314,19 +358,22 @@ void fstk_RunInclude(char const *path)
 		free(fullPath);
 		if (generatedMissingIncludes) {
 			if (verbose)
-				printf("Aborting (-MG) on INCLUDE file '%s' (%s)\n",
-				       path, strerror(errno));
+				printf("Aborting (-MG) on %s file '%s' (%s)\n",
+				       fromArgv ? "`-I`" : "INCLUDE", path,
```

Comment:
I'd like to settle on single error messages that work well for both possibilities. Then instead of one `fstk_RunInclude` function with a boolean flag, we could have `fstk_RunInclude` and `runIncludeFile`. `fstk_RunInclude` would call `runInclude` and then `lexer_SetStateAtEOL`; `runIncludeFile` would call `runInclude` and then `lexer_SetState`.

Reply:
I think "included file" would work for the error messages:
- `Aborting (-MG) on included file '%s' (%s)`
- `Unable to open included file '%s': %s`
- `Failed to alloc file info for included file: %s`

Reply:
`-I` files should not be subject to `-MG`, as they are specified externally. They shouldn't be subject to `-i` search either.

Reply:
gcc does make its `-include` files subject to the `-I` directories and to `-MG`, which I think is useful behavior.